package.json

```diff
@@ -4,13 +4,17 @@
   "description": "A slow down plugin for fastify",
   "main": "index.js",
   "type": "module",
+  "types": "index.d.ts",
   "files": [
     "index.js",
+    "index.d.ts",
     "lib"
   ],
   "scripts": {
     "lint": "eslint .",
-    "test": "c8 --100 tap -J --no-coverage test/*.test.js",
+    "test": "npm run test:unit && npm run test:types",
```

Comment:
What if we concurrently to do this?

Reply:
it's a fairly common pattern to do it this way, and little benefit doing it in parallel since both of these are fairly quick so we can avoid installing yet another package